.github/CODEOWNERS

```diff
@@ -0,0 +1,90 @@
+# Lists some code owners.
+#
+# A codeowner just oversees some part of the codebase. If an owned file is changed then the
+# corresponding codeowner receives a review request. An approval of the codeowner might be
+# required for merging a PR (depends on repository settings).
+#
+# For details about syntax, see:
+# https://help.github.com/en/articles/about-code-owners
+# But here are some important notes:
+#
+# - Glob syntax is git-like, e.g. `/core` means the core directory in the root, unlike `core`
+#   which can be everywhere.
+# - Multiple owners are supported.
+# - Either handle (e.g, @github_user or @github/team) or email can be used. Keep in mind,
+#   that handles might work better because they are more recognizable on GitHub
+#   and you can use them for mentioning unlike an email.
+# - The latest matching rule, if multiple, takes precedence.
+
+# Top levels
+/.cargo @paritytech/ci
+/.config @paritytech/ci
+/.github @paritytech/ci @paritytech/release-engineering
+/.gitlab @paritytech/ci
+/bridges @paritytech/bridges-core
+/cumulus @paritytech/cumulus-locks-review @paritytech/core-devs
+/docker @paritytech/ci @paritytech/release-engineering
+/docs @paritytech/docs-audit
+/polkadot @paritytech/polkadot-review
```

Comment:
hmm, I think this was previously only needed for runtime files, which are going to be migrated to https://github.com/polkadot-fellows/runtimes

https://github.com/paritytech/polkadot-sdk/pull/1239#issuecomment-1697070397

Reply:
Are you suggesting using `core-devs` for `/polkadot` as catch-all as well ?

Reply:
yeah, probably start with not to strict rules and enforce other rules via PCR

Reply:
Yes the idea here is not to be too strict but have "full" coverage for the relevant files (ie all files but things like `.gitignore`, etc...)

Reply:
This is definitely too strict